phoenix-core-server/src/main/java/org/apache/phoenix/mapreduce/AbstractBulkLoadTool.java

```diff
@@ -249,7 +250,7 @@ private int loadData(Configuration conf, CommandLine cmdLine) throws Exception {
         if (cmdLine.hasOption(OUTPUT_PATH_OPT.getOpt())) {
             outputPath = new Path(cmdLine.getOptionValue(OUTPUT_PATH_OPT.getOpt()));
         } else {
-            outputPath = new Path("/tmp/" + UUID.randomUUID());
+            outputPath = new Path(tmpPath + "/" + UUID.randomUUID());
```

Comment:
Please check EACH PATH manually.
This is another obvious HadoopFS path, the type is org.apache.hadoop.fs.Path. It is defined just a few lines above.